docs/b2500.md

```
@@ -1,390 +1,789 @@
-md=0-500, For example, setting 300: cd=06, md=300
 
-## 6. Timed and fixed power discharge settings(Flash storage)
+---
+
+#### Discharge mode / output control (V1 only)
 
-### 6.1 Public
+Controls which output ports are enabled. The `md` value is a bitmask: bit 0 = OUT1, bit 1 = OUT2.
 
-**Topic:**
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)